### PR TITLE
[MINI-3451] Addition of a banner message

### PR DESCRIPTION
**Pull request by Climbatize**
# Description

Addition of a banner message when sending a message to contacts
requires https://github.com/rakutentech/js-miniapp/pull/142

https://user-images.githubusercontent.com/1507391/125053525-2dc34480-e0e0-11eb-973f-25b13b8e23dd.mp4

## Links
[MINI-3451](https://jira.rakuten-it.com/jira/browse/MINI-3451)

# Checklist
- [X] I have read the [contributing guidelines](CONTRIBUTING.md).
- [X] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [ ] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by khairul-alam-licon — APPROVED** [review]
LGTM

File: Example/Utils/UIColor+MiniApp.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
+//
+//  UIColor+MiniApp.swift
+//  MiniApp_Example
+//
+//  Created by Villa, Pierre | Pm | MTSD on 2021/07/09.
+//  Copyright © 2021 Rakuten Group, Inc. All rights reserved.
+//
```
Comment (rleojoseph) [inline Example/Utils/UIColor+MiniApp.swift:7]: You can remove them

**Review by rleojoseph — COMMENTED** [review]
Looks good to me :+1: Just one comment.

Also, if the banner message is too large, do you think it's okay to truncate tail?

**Review by rleojoseph — APPROVED** [review]

**Comment by rleojoseph** [comment]
I just saw the character limitation in JS PR. Ignore my comment about the truncate tail

**Comment by Climbatize** [comment]
> Looks good to me 👍 Just one comment.
> 
> Also, if the banner message is too large, do you think it's okay to truncate tail?

Thanks for the review!
Truncating is done on JS side, banner message is 128 characters max ;)

File: Example/Utils/UIColor+MiniApp.swift (hunk 1 above)
Comment (Climbatize) [reply Example/Utils/UIColor+MiniApp.swift]: thanks,I missed it :)

**Pull request merged**